Editor/InputSystemChecker.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using Innoactive.Creator.Core.Input;
+using UnityEditor;
+
+[InitializeOnLoad]
+public static class InputSystemChecker
+{
+    private const string message =
+        "From Creator version 2.11 onwards Unity's new Input System is required but not added to your project yet." +
+        "\n\nTo switch from the legacy input system to the new one, open the 'Player Settings' and set the " +
+        "option 'Active Input Handling' to 'Both' or 'Input System Package (New)'.";
+
+    /// <summary>
+    /// This is a check if the new input system is active OR another concrete implementation of the InputController exists.
+    /// </summary>
+    static InputSystemChecker()
+    {
+        try
+        {
+            // This will throw an InvalidOperationException when no concrete implementation is found.
+            Type type = InputController.ConcreteType;
+        }
+        catch (InvalidOperationException)
+        {
+            if (CreatorProjectSettings.Load().IsFirstTimeStarted == false)
+            {
+                EditorUtility.DisplayDialog("Attention required!", message, "Understood");
```

Comment:
Once I've dismissed this warning, I will never see it again until next Editor launch? Or will it reoccur after every compile?

Reply:
EVERY compile

Reply:
As the whole creator will break on play, this is intended to be annoying.